pom.xml

```diff
@@ -48,13 +50,101 @@
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>flatten-maven-plugin</artifactId>
+        <version>1.3.0</version>
```

Comment:
Nice catch.
I need to update this in `maven-parent` here:
https://github.com/devonfw/maven-parent/blob/ef1dcfa4d6280f3ec6d31ef360b697dbcacc8721/pom.xml#L49